tcf_website/views/review.py

```diff
@@ -83,7 +73,32 @@ def new_review(request):
     return render(request, 'reviews/new_review.html')
 
 
+@login_required()
+def check_duplicate(request):
+    """Check for duplicate reviews when a user submits a review
+    based on if it's the same course with the same instructor/semester.
+    Used for an ajax request in new_review.html"""
+
+    form = ReviewForm(request.POST)
+    instance = form.save(commit=False)
+
+    reviews_on_same_class = request.user.review_set.filter(
+        course=instance.course,
+        instructor=instance.instructor,
+        semester=instance.semester
+    )
+
+    # Review already exists so it's a duplicate; inform user
+    if reviews_on_same_class.exists():
+        response = {"duplicate": True}
+        return JsonResponse(response)
+
+    response = {"duplicate": False}
+    return JsonResponse(response)
+
 # Note: Class-based views can't use the @login_required decorator
+
```

Comment:
What's with these extra whitespace lines? I think this comment is supposed to be directly above `DeleteReview`?